fair/templates/config.jinja

```diff
@@ -1,7 +1,10 @@
 run_metadata:
   description: Project description
```

Comment:
I don't know whether this is the right place to put this comment in, but given this is a dummy entry in the yaml file, I wonder if it makes sense to put this line at the bottom just before the `script` key. I might move the `default_input_namespace` and `default_output_namespace` down as well (but above `description`), as they could possibly be changed. The rest are probably never going to be edited.